settings.js

```diff
@@ -144,7 +144,7 @@ window.spinnakerSettings = {
     jobs: false,
     snapshots: false,
     travis: false,
-    pipelineTemplates: false,
+    pipelineTemplates: true,
```

Comment:
Not sure this should be a part of the PR

Reply:
oh my, sorry for this :) I will correct it

Reply:
Done